README.md

```diff
@@ -82,10 +82,9 @@ contribute in some way.
   Documentation powered by Netlify.
 </p>
 
-### Contributing
+###  For Contribution
```

Comment:
No difference.

Reply:
it shows manual you have to follow for contribution contributing does not give any meaning